### PR TITLE
[UI] Migrate CustomAvatar and ErrorBoundary components to MUI 5 using  Sistent

### PR DESCRIPTION
**Pull request by Vaibhav701161**
This PR migrates the CustomAvatar and ErrorBoundary components to MUI 5 while using Sistent for design tokens and component styling consistency.

**Note for reviewers:-** 
Reviewers, please verify that the MUI 5 migration and Sistent integration have been implemented correctly and that the components behave as expected.

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Divyansh200102** [comment]
Hi @Vaibhav701161 i am working on a similar pr and getting this error in `Meshery Snapshot Service / MeshMapScreenshot (pull_request_target) ` this check

`Download action repository 'actions/checkout@v4' (SHA:eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871)
Download action repository 'layer5labs/Kanvas-Snapshot@v0.2.14' (SHA:5ce3576[25](https://github.com/meshery/meshery/actions/runs/11391563481/job/31695628326?pr=12427#step:1:29)b133fd3e2739d423a7d4f4cd3f6712c)
Error: layer5labs/Kanvas-Snapshot/v0.2.14/action.yml: (Line: 25, Col: 79, Idx: 895) - (Line: 25, Col: 140, Idx: 956): While parsing a block mapping, did not find expected key.
Error: System.ArgumentException: Unexpected type '' encountered while reading 'action manifest root'. The type 'MappingToken' was expected.
   at GitHub.DistributedTask.ObjectTemplating.Tokens.TemplateTokenExtensions.AssertMapping(TemplateToken value, String objectDescription)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Failed to load layer5labs/Kanvas-Snapshot/v0.2.14/action.yml`

Do you have any idea on how to fix this?

**Comment by vishalvivekm** [comment]
@Vaibhav701161 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT | 7:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

File: ui/components/ErrorBoundary.js [hunk 1]
```diff
@@ -1,41 +1,37 @@
...
-        </div>
-      );
-    }
-    return this.props.children;
-  }
-}
+  return (
+    <UsesSistent>
+      <SistentErrorBoundary
+        fallback={({ error, resetErrorBoundary }) => (
```
Comment (ianrwhitney) [inline ui/components/ErrorBoundary.js:15]: Can you test this out in the UI and add some screenshots? Ths sisetent error boundary take customFallback vs fallback as prop. Let me know if im mistaken. Also curious if we need a custom fallback vs using generic. thanks 🚀

**Review by ianrwhitney — COMMENTED** [review]
Overall, looks good. A few comments for your consideration. Thank you for contributions. 🚀

**Comment by sudhanshutech** [comment]
@Vaibhav701161 only just update your changes for avatar component since error boundary already done

**Comment by sudhanshutech** [comment]
@Vaibhav701161 checking in again...any update? are you still with us?

**Pull request closed without merging**